Explorer/Assets/Scripts/ECS/StreamableLoading/Common/Systems/LoadSystemBase.cs

```diff
@@ -55,6 +55,12 @@ public override void Initialize()
             cancellationTokenSource = new CancellationTokenSource();
         }
 
+        public LoadSystemBase<TAsset, TIntention> InitializeAndReturnSelf()
```

Comment:
Let's move it to the Demo domain as an extension for example. This method looks strange and should not be used in the normal flow